Swift.gitignore

```diff
@@ -31,3 +31,11 @@ DerivedData
 # Carthage/Checkouts
 
 Carthage/Build
+
+# Libraries
+#
+# Swift libraries built as artifacts of part of a Swift project
+
+*.swiftmodule
+*.swiftdoc
+*.dylib
```

Comment:
While the other two rules are very specific to Swift, this third one seems to be more general. Is there a particular reason why we're needing to introduce this now?